src/Microsoft.Identity.Client/MsalServiceException.cs

```diff
@@ -237,7 +237,14 @@ public MsalServiceException(string errorCode, string errorMessage, int statusCod
         /// <returns>A string representation of the current exception.</returns>
         public override string ToString()
         {
-            return base.ToString() + string.Format(CultureInfo.InvariantCulture, "\n\tStatusCode: {0}\n\tClaims: {1}", StatusCode, Claims);
+            return base.ToString() + string.Format(
```

Comment:
>return [](start = 12, length = 6)

what about SubError, Error, CorrelationId and ? and errorcode ?

Reply:
They are all in the ResponseBody as a json, it's pretty readable as is. Actually we also don't need claims.

Reply:
base.ToString() will display error code and error message. SubError, CorrelationID etc. are contained in the ResponseBody, and double logging creates confusion. The unit test does assert all of these are present.